### PR TITLE
Add: Maintainers Contributing Guide

### PR DESCRIPTION
**Pull request by sarah11918**
[X] New or updated content

This adds a new page MAINTAINERS.md - A Contributing To the Docs Guide specifically for Astro maintainers.

This is an initial draft page, but is ready for all your edits, comments and reviews! As soon as something is good enough to publish, it can be merged and the document can receive ongoing improvements.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+- `improve existing documentation` is used for smaller, but non-trivial documentation improvements to items that are currently documented. This distinction is helpful because we have already considered this feature as part of the entire Docs site, and the change might be small enough to be localized. For example, improving the wording of an instruction or adding an extra example to illustrate a particular nuance of a topic that we have already documented.
+
+- `site improvement` is a broad category, but helps us distinguish content and writing tasks from site issues and maintanance. This distinction is helpful to allow community members and maintainers to easily filter tasks to either focus on or ignore content writing Issues and PRs. This allows community members (and maintainers!) more easily contribute according to their own skills and interests.
```
Comment (delucis) [inline MAINTAINERS.md:30]: Although it could be inferred, this wasn’t 100% clear which of the two things (site issues vs content) this label was for. Maybe reword like this?

```suggestion
- `site improvement` is a broad category, but is used to label site issues and maintenance tasks as opposed to content and writing tasks. This distinction is helpful to allow community members and maintainers to easily filter tasks to either focus on or ignore content writing Issues and PRs. This allows community members (and maintainers!) more easily contribute according to their own skills and interests.
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- Navigate to several pages, including following anchor links, to test behaviour
+- Conduct any a11y checks or tests you have access to: screen reader behaviour, colour contrast ratios, etc.
+- For most non-trival PRs, the Docs Infrastructure Lead will likely want to do a final review themselves before the PR is merged.
+
+### Language Translations
+
+For PRs that are translations to existing Docs content, including new page addtions as well as smaller updates and corrections:
+
+- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the transators' process
+- If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! :partying_face
```
Comment (delucis) [inline MAINTAINERS.md:79]:
```suggestion
- If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! 🥳
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides loctaed in their language folder within `/src/i18n/`
+- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PRs Netlify deploy preview to verify that nothing is visually out of place.
+
+
+## Submitting your own PRs
+
+This section is much shorter now that we are protecting the `main` branch of the Astro repos!  :sweat_smile
```
Comment (delucis) [inline MAINTAINERS.md:86]:
```suggestion
This section is much shorter now that we are protecting the `main` branch of the Astro repos! 😅
```

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- For most non-trival PRs, the Docs Infrastructure Lead will likely want to do a final review themselves before the PR is merged.
+
+### Language Translations
+
+For PRs that are translations to existing Docs content, including new page addtions as well as smaller updates and corrections:
+
+- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the transators' process
+- If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! :partying_face
+- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides loctaed in their language folder within `/src/i18n/`
+- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PRs Netlify deploy preview to verify that nothing is visually out of place.
```
Comment (delucis) [inline MAINTAINERS.md:81]:
```suggestion
- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place.
```

**Review by delucis — COMMENTED** [review]
Thanks for tackling this Sarah! I spotted a few minor things, but mostly this seems spot on 🙌

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+Providing friendly, helpful, timely feedback is important to maintaining our community. It shows respect and appreciation for their time and contribution.
+
+As a maintainer, helpful things you can do:
+
+- Leave a "Thank you for this!" message, or a simple emoji reaction, in an Issue or a PR if an Issue or PR hasn't had any activity yet. Similarly, "Just checking in!" or "We haven't forgotten about this, but are waiting on..." messages are also friendly and helpful!
+
+- Ask any obvious clarifying questions, even if you do not intend to follow up with or take responsibility for the PR itself. Anticipate what further information might be useful to include, or address any weakness or points of confusion.
+
+- Give general feedback (without commiting to responsibility for merging). "I can reproduce this problem!" or "I agree, this code example isn't working as written," save others time when trying to evaluate PRs.
```
Comment (Yan-Thomas) [inline MAINTAINERS.md:47]: I'm unsure if this is because English isn't my native language, but "(without commiting to responsibility for merging)" sounds a bit off to me as a whole, but the real problem is this small typo:
```suggestion
- Give general feedback (without committing to responsibility for merging). "I can reproduce this problem!" or "I agree, this code example isn't working as written," save others time when trying to evaluate PRs.
```

File: MAINTAINERS.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- Conduct any a11y checks or tests you have access to: screen reader behaviour, colour contrast ratios, etc.
+- For most non-trival PRs, the Docs Infrastructure Lead will likely want to do a final review themselves before the PR is merged.
+
+### Language Translations
+
+For PRs that are translations to existing Docs content, including new page addtions as well as smaller updates and corrections:
+
+- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the transators' process
+- If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! :partying_face
+- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides loctaed in their language folder within `/src/i18n/`
```
Comment (Yan-Thomas) [inline MAINTAINERS.md:80]:
```suggestion
- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for your amazing work @sarah11918! Just found a few typos.

File: MAINTAINERS.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+Providing friendly, helpful, timely feedback is important to maintaining our community. It shows respect and appreciation for their time and contribution.
+
+As a maintainer, helpful things you can do:
+
+- Leave a "Thank you for this!" message, or a simple emoji reaction, in an Issue or a PR if an Issue or PR hasn't had any activity yet. Similarly, "Just checking in!" or "We haven't forgotten about this, but are waiting on..." messages are also friendly and helpful!
+
+- Ask any obvious clarifying questions, even if you do not intend to follow up with or take responsibility for the PR itself. Anticipate what further information might be useful to include, or address any weakness or points of confusion.
+
+- Give general feedback (without committing to responsibility for merging). "I can reproduce this problem!" or "I agree, this code example isn't working as written," save others time when trying to evaluate PRs.
```
Comment (sarah11918) [inline MAINTAINERS.md:47]:
```suggestion
- Give general feedback or opinions, even if you do not intend to follow up with or take responsibility for the PR itself. Comments such as "I can reproduce this problem!" or "I agree, this code example isn't working as written," save others time when trying to evaluate PRs.
```
I reused the same wording as the point above!

**Comment by delucis** [comment]
LGTM! Thanks @sarah11918 🚀
